test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExtendedQueryTagTests.cs

```diff
@@ -200,6 +217,26 @@ public async Task GivenInvalidTagLevelInRequestBody_WhenCallingPostAync_ThenShou
             Assert.Equal("The field '[0].Level' in request body is invalid: Input Dicom Tag Level 'Studys' is invalid. It must have value 'Study', 'Series' or 'Instance'.", response.Content.ReadAsStringAsync().Result);
         }
 
+        private async Task DeleteExtendedQueryTagIfExists(DicomTag tag)
+        {
+            // Try to delete this extended query tag if it exists.
+            await _tagManager.DeleteExtendedQueryTagAsync(string.Concat(tag.Group.ToString("D4"), tag.Element.ToString("D4")));
+        }
+
+        private static DicomTag RandomDicomTag(string privateCreator = null)
```

Comment:
For now, I am reverting this change and not creating any other private tag. I will see if just deleting the existing extended query tag would do the trick. If not, we can fall back to this.
But yes, as for creation of dicom tags, since any generated number is not necessarily a valid dicom tag, I agree with the proposition.